shopify_app.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.2.4"
 
   s.add_runtime_dependency('rails', '>= 4.2.6')
-  s.add_runtime_dependency('shopify_api', '~> 4.2.2')
+  s.add_runtime_dependency('shopify_api', '>= 4.2.2', '< 5.0')
```

Comment:
Good question. I added it because before 13d92a8 it was `~> 4.2`, which means `>= 4.2.0 && < 5.0`. If you aren't too worried about breaking changes in the next major version of `shopify_api`, then I'd be happy to remove it.